README.md

```diff
@@ -205,3 +205,54 @@ Add new dependencies to `Pipfile`, run:
     pipenv install <name>
 
 and then commit the changes to `Pipfile` and `Pipfile.lock`.
+
+
+
+
+### Configuring your local development environment
+#### Prerequisites
+This tutorial assumes you have the following dependencies already installed.
+* Git
+* Python
+* PostgreSQL 
+
+1. Clone the git repo
+```sh
+git clone git@github.com:seattleflu/id3c.git
+```
+
+2. To fill the `seattleflu` database with the backup db copy, run 
+`pg_restore --data-only --single-transaction --schema receiving --table enrollment --dbname seattleflu seattleflu-production-<date>.pgdb`
+
+3. To restore the presence/absence data on your local dev instance: 
+`pg_restore --dbname seattleflu presence_absence.pgdb`
+
+
+#### Prevent wrapping within psql
+Set pager environment variable to `less` and specify which `less` method to use. If there's less than a screen full of information, don't page it.
+```sh
+PAGER=less LESS=SFRXi psql seattleflu
+```
+
+#### Psql
+Reset the processing log for all rows to be blank. When the enrollments pipeline is run, it will process all rows with blank processing logs for the current revision. These rows will be marked `for update`. 
+```sql
+update receiving.enrollment set processing_log = '[]';
+```
```

Comment:
This might be another candidate for adding to a `DEVELOPMENT.md` file with common tasks.